integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -6662,6 +6662,14 @@ msgstr "Neue Nachricht verfassen"
 msgid "Save & Send"
 msgstr "Speichern & Senden"
 
+#: cms/templates/push_notifications/push_notification_form.html
+msgid ""
+"Managers for one but not all selected regions will be able to view, but not "
```

Comment:
The sentence is pretty hard to understand. Do I understand correctly that the meaning is the following:
`Any manager with access to one of the selected regions can view but not edit this news message.`

Reply:
Yes. The clearest would be to define it mathematically, but there is no way to make that sound good :cry: 

$\forall\ \text{Manager}\ m \in M |\ \text{regions for a News Message}\ N_{regions} \setminus m_{regions} \neq \emptyset \Longrightarrow \overline{\text{editable}}$

Reply:
> Any manager with access to one of the selected regions can view but not edit this news message.

If this is an alternate suggestion, I'm fine with it, if we don't make it sound like I also cannot edit it if I have access to all regions:

> Any manager with access to only some of the selected regions can view but not edit this news message.